### PR TITLE
Modify test_positive_assign_ansible_role_variable_on_host to verify roles

### PR DESCRIPTION
**Pull request by Gauravtalreja1**
### Problem Statement
As per suggestions provided in component audit-2, we need to add a test to verify roles assignment with variables on host, and also cover multiple roles with webUI

### Solution
We had this test to verify roles assignment with variables on host, just modifying the existing test to cover multiple roles, and verifying roles assigned on details page with few more changes

### CODE REVIEW

**Comment by Gauravtalreja1** [comment]
trigger: test-robottelo
pytest: tests/foreman/ui/test_ansible.py::TestAnsibleCfgMgmt::test_positive_assign_ansible_role_variable_on_host

**Comment by Satellite-QE** [comment]
**PRT Result**
```
Build Number: 7469
Build Status: UNSTABLE
PRT Comment: pytest tests/foreman/ui/test_ansible.py::TestAnsibleCfgMgmt::test_positive_assign_ansible_role_variable_on_host --external-logging
Test Result : ============= 1 passed, 12 warnings, 1 error in 838.85s (0:13:58) ==============
```

**Comment by Gauravtalreja1** [comment]
trigger: test-robottelo
pytest: tests/foreman/ui/test_ansible.py::TestAnsibleCfgMgmt::test_positive_assign_ansible_role_variable_on_host

**Comment by Satellite-QE** [comment]
**PRT Result**
```
Build Number: 7470
Build Status: SUCCESS
PRT Comment: pytest tests/foreman/ui/test_ansible.py::TestAnsibleCfgMgmt::test_positive_assign_ansible_role_variable_on_host --external-logging
Test Result : ================== 1 passed, 12 warnings in 892.08s (0:14:52) ==================
```

File: tests/foreman/ui/test_ansible.py [hunk 1]
```diff
@@ -165,15 +165,16 @@ def test_positive_assign_ansible_role_variable_on_host(
...
-            assert f'Ansible role [{SELECTED_ROLE}] was deleted.' in result[0]['message']
+            result = target_sat.cli.Ansible.variables_delete({'name': key})
+            assert f'Ansible variable [{key}] was deleted.' in result[0]['message']
+            for role in SELECTED_ROLE:
+                result = target_sat.cli.Ansible.roles_delete({'name': role})
+                assert f'Ansible role [{role}] was deleted.' in result[0]['message']
 
         key = gen_string('alpha')
-        SELECTED_ROLE = 'redhat.satellite.activation_keys'
+        SELECTED_ROLE = ['redhat.satellite.activation_keys', 'RedHatInsights.insights-client']
```
Comment (shweta83) [inline tests/foreman/ui/test_ansible.py:175]: Any reason to add additional role here?

Comment (shweta83) [reply tests/foreman/ui/test_ansible.py:175]: Is there any BZ for it?

**Review by shweta83 — APPROVED** [review]
All looks good, just one pending question.

File: tests/foreman/ui/test_ansible.py (hunk 1 above)
Comment (Gauravtalreja1) [reply tests/foreman/ui/test_ansible.py:175]: no, it was asked in component audit?

**Review by shubhamsg199 — APPROVED** [review]
Ack

**Pull request merged**